test/mysql/sql_warning.txt

```diff
@@ -0,0 +1 @@
+Syntax error found near Column Identifier (SELECT Results Clause): SELECT 'Connection.query without a callback
```

Comment:
I think you added this file accidentally (it's created by the indexer and it's easy to miss).

Reply:
Just dropped it and pushed again, thank you.